### ISSUE TITLE
[EN][QUEST/DROP][Hellfire Ramparts] Dark Tidings

### ISSUE THREAD

**Issue by ghost** [Atlantiss/BugTracker#6976]
The item "Ominous Letter" should drop from Vazruden in Hellfire Ramparts but currently does not. http://wowwiki.wikia.com/wiki/Ominous_Letter

It will start the quest "Dark Tidings" http://www.wowhead.com/quest=9588/dark-tidings, which after completion will unlock the quest "The Blood is Life" http://www.wowhead.com/quest=9590/the-blood-is-life.

Currently the quest "The Blood is Life" is already open to level appropriate players without doing the prerequisite "Dark Tidings", which is started by the Ominous Letter, that does not currently drop in the game like it is supposed to.

**Comment by Navedonar** [comment]
Don't believe this drops anymore - killed twice (once at 59 and once at 50). No drops, and it used to be 100%. Most likely removed with the shattering or the introduction of the new dungeon quests they added some patches back in TBC dungeons.
-1
Soeroah on 2012/05/20 (Patch 4.3.4)
I could have sworn I got it on my Priest a few weeks ago, but I just ran the dungeon twice on my Hunter and no luck. I reckon it was removed in that Northrend/Outland patch that moved quests inside the dungeons and got rid of early group quests in the continents.
